### PR TITLE
Update de_de.json

### PR DESCRIPTION
**Pull request by Wizqok**
matching en_us now

### CODE REVIEW

File: shared/src/main/resources/assets/farmingforblockheads/lang/de_de.json [hunk 1]
```diff
@@ -1,12 +1,53 @@
...
+  "item.farmingforblockheads.red_fertilizer": "Roter Dünger",
+  "item.farmingforblockheads.green_fertilizer": "Grüner Dünger",
+  "item.farmingforblockheads.yellow_fertilizer": "Gelber Dünger",
   "itemGroup.farmingforblockheads.farmingforblockheads": "Farming for Blockheads",
   "container.farmingforblockheads.market": "Markt",
+  "commands.farmingforblockheads:reload.success": "Farming for Blockheads-Register neu geladen.",
   "gui.farmingforblockheads:market.no_selection": "Keine Auswahl",
   "gui.farmingforblockheads:market.tooltip_cost": "Kosten: %s",
   "gui.farmingforblockheads:market.tooltip_seeds": "Samen",
-  "gui.farmingforblockheads:market.tooltip_saplings": "Setzlinge",
```
Comment (BlayTheNinth) [inline shared/src/main/resources/assets/farmingforblockheads/lang/de_de.json:8]: Setzlinge is the vanilla translation for saplings

File: shared/src/main/resources/assets/farmingforblockheads/lang/de_de.json [hunk 2]
```diff
@@ -1,12 +1,53 @@
...
-  "gui.farmingforblockheads:market.tooltip_other": "Sonstiges",
```
Comment (BlayTheNinth) [inline shared/src/main/resources/assets/farmingforblockheads/lang/de_de.json:9]: Sonstiges seems more fitting than Andere

**Review by BlayTheNinth — CHANGES_REQUESTED** [review]

**Review by Wizqok — COMMENTED** [review]
changed

**Comment by Wizqok** [comment]
are you gonna push this also for other branches? 
Br

**Comment by BlayTheNinth** [comment]
Language updates are released alongside regular updates, so only versions that receive updates from hereon out will include these changes

**Pull request merged**